languages/typescript/code/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@codecrafters/{{course-slug}}",
+  "description": "{{course-name}} challenge, from CodeCrafters",
+  "type": "module",
+  "scripts": {
+    "dev": "bun run app/main.ts"
+  },
+  "dependencies": {
+    "async": "^3.2.0"
```

Comment:
Don't think this is required for the template (only specific challenges will need this)